src/domUpdates.js

```diff
@@ -87,6 +88,21 @@ const domUpdates = {
     } else {
       upcomingTripSection.innerText = 'Book a Trip!'
     }
+  },
+
+// apendChild?
+  destinationDropdown(destinationsData) {
+    let selectDestination = document.querySelector('#destinations');
+    this.allDestinations.forEach(destination => {
+      let destinationOption = `<option id="${destination.id}">${destination.destination}</option>`;
```

Comment:
is this currently working?
because it seems that you need to use the `+=` to add all the options